tests/fixtures/encoder_decoder/nlvr_coverage_semantic_parser/mml_init_experiment.json

```diff
@@ -0,0 +1,68 @@
+{
+  "dataset_reader": {
+    "type": "nlvr",
+    "output_agendas": true
+  },
+  "vocabulary": {
+    "non_padded_namespaces": ["denotations"]
```

Comment:
This overrides the `["*tags", "*labels"]` default, so `rule_labels` will actually get padding if you do this.

Reply:
I'm not sure what you mean. Is this regarding the change to `ModelTestCase`?

Reply:
This is in general. If you override this value, you remove the defaults, so they need to be added back in.

Reply:
Thanks! That's good to know.